src/cargo/core/compiler/compilation.rs

```diff
@@ -73,8 +73,14 @@ pub struct Compilation<'cfg> {
     pub target: String,
 
     config: &'cfg Config,
+
+    // Rustc Wrapper Combinations
```

Comment:
The comments here should probably be doc-comments, and could use some elaboration on what exactly the mean.